src/blocks/PressCardBlock/PressCardItem.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react'
+import styled from 'react-emotion'
+
+import { colors } from '@hedviginsurance/brand'
+import { PressCardItemProps } from '.'
+import { MOBILE_BP_DOWN } from '../../components/blockHelpers'
+import { DeferredImage } from '../../components/DeferredImage'
+import { getStoryblokImage } from '../../utils/storyblok'
+
+const PressItemContainer = styled('div')({
+  borderRadius: 8,
+  padding: 24,
+  margin: '0 20px',
+  maxWidth: 330,
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'center',
+  justifyContent: 'center',
+  backgroundColor: 'white',
```

Comment:
should this always be white, no matter what the block background is?

Reply:
Is off white now when block background is white